app/services/promotional_tweet/post_promotional_tweet.rb

```diff
@@ -0,0 +1,42 @@
+class PostPromotionalTweet
+
+  def initialize(promotional_tweet, image, brand)
+    @message = promotional_tweet.message
+    @image   = image
+    @brand   = brand
+    @client  = brand.twitter_rest_client
+  end
+
+  def send!
+    @tweet = post_promo_tweet
```

Comment:
Posting to Twitter could also raise an error. I don't think we need to anything here. Just wanted to bring this up so you were aware.